shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/container/config/proxy/ScalingProxyClusterContainerConfigurationFactory.java

```diff
@@ -20,31 +20,33 @@
 import lombok.RequiredArgsConstructor;
 import org.apache.shardingsphere.infra.database.type.DatabaseType;
 import org.apache.shardingsphere.test.integration.env.container.atomic.adapter.config.AdaptorContainerConfiguration;
+import org.apache.shardingsphere.test.integration.env.container.atomic.adapter.config.ProxyClusterContainerConfigurationFactory;
 import org.apache.shardingsphere.test.integration.env.container.atomic.util.DatabaseTypeUtil;
 
-import java.util.Collections;
-import java.util.HashMap;
 import java.util.Map;
 
 /**
- * Scaling proxy cluster container configuration.
+ * Scaling proxy cluster container configuration creator.
  */
 @RequiredArgsConstructor
```

Comment:
What is the useful of @RequiredArgsConstructor?

Reply:
fixed